commit 6a2f0efda5eabd590d35a68e09218ee74dc29030
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-07-25T08:22:03-07:00 (2 months ago)
    
    TRITON-1835 cloudapi migration tests do not need to check the sync message field
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/test/machines/migrations.js b/test/machines/migrations.js
index 84fb0004c..9239dd9fc 100644
--- a/test/machines/migrations.js
+++ b/test/machines/migrations.js
@@ -194,7 +194,6 @@ module.exports = function migrate(suite, client, other, machine, callback) {
                 });
                 progressEvents.forEach(function testPrgrEvt(evt) {
                     t.ok(evt.current_progress, 'Should have current_progress');
-                    t.ok(evt.message, 'Should have a message');
                     t.ok(evt.state, 'Should have event state');
                     t.ok(evt.total_progress, 'Should have total_progress');
                     if (evt.started_timestamp) {
